src/Network/Network/Common/NetworkResourceManagerProfile.cs

```diff
@@ -644,6 +644,22 @@ private static void Initialize()
                 // CNM to MNM
                 cfg.CreateMap<CNM.PSBackendAddressPool, MNM.BackendAddressPool>();
 
+                // LoadBalancerBackendAddress
+                // CNM to MNM
+                cfg.CreateMap<CNM.PSLoadBalancerBackendAddress, MNM.LoadBalancerBackendAddress>();
+
+                // NetworkInterfaceIPConfiguration
+                // MNM to CNM
+                cfg.CreateMap<MNM.NetworkInterfaceIPConfiguration, CNM.PSNetworkInterfaceIPConfiguration>();
+
+                // NetworkInterfaceIPConfiguration
+                // CNM to MNM
+                cfg.CreateMap<CNM.PSNetworkInterfaceIPConfiguration, MNM.NetworkInterfaceIPConfiguration>();
+
+                // LoadBalancerBackendAddress
```

Comment:
move above nic?

Reply:
I should actually remove NetworkInterfaceIPConfiguration conversion here as we are not allowing that in this change.